lib/services/elasticsearch.js

```diff
@@ -1031,6 +1031,26 @@ class ElasticSearch extends Service {
       .catch(error => this._esWrapper.reject(error));
   }
 
+  /**
+   * Deletes a collection
+   *
+   * @param {String} index - Index name
+   * @param {String} collection - Collection name
+   *
+   * @returns {Promise}
+   */
+  deleteCollection (index, collection) {
+    const esRequest = {
+      index: this._getESIndex(index, collection)
+    };
+
+    return this._client.indices.delete(esRequest)
+      .then(() => {
+        // return nothing
```

Comment:
I think we should return something like the collection name. As it's made for https://docs.kuzzle.io/core/1/api/controllers/document/delete/

Reply:
IMHO unnecessary returns should be avoided, in this case I don't see the point on returning the collection name we just deleted.
Maybe something like the collection actual mapping could be useful but again without concrete usecase I didn't think it's a good idea to add complexity in our API.
What do you think ?

Reply:
Our current standard for acknowledged requests, but possibly not yet executed, is to return `{ "acknowledge": true }`

Here, if I understand correctly, we don't return until after the collection is deleted. So we could either return the acknowledgement message above, or simply `"OK"`, à la Redis.

I think we shouldn't leave the "result" part null though: it can be puzzling for clients who are trying to decide if a response like that is an error or a success: `{error: null, result: null}`

Reply:
I don't think this is useful to return anything here. But the custom is to return an explicit `null` value if we do want to return nothing.